esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -1,15 +1,15 @@
 //*******************************************************
-// mLRS WiFi Bridge
+// mLRS Wireless Bridge
 // Copyright (c) www.olliw.eu, OlliW, OlliW42
 // License: GPL v3
 // https://www.gnu.org/licenses/gpl-3.0.de.html
 //*******************************************************
-// Basic but effective & reliable transparent WiFi<->serial bridge
+// Basic but effective & reliable transparent WiFi and Bluetooth<->serial bridge
 //*******************************************************
 // 22. Aug. 2023
 //*********************************************************/
 // inspired by examples from Arduino
-// ArduinoIDE 2.0.3, esp32 by Espressif Systems 2.0.6
+// ArduinoIDE 2.1.1, esp32 by Espressif Systems 2.0.11
```

Comment:
is this just what you happen to have or are these in fact the minimal versions required?

Reply:
This is what I used to test.

Reply:
ok, fair enough. Thx for the clarification. I will test too, and when we will see :)

Reply:
it compiles and works partially (see below). So this version change is not needed and can be reverted.

Reply:
Great, thanks. Reverted.